ACTPlugin.cs

```diff
@@ -48,14 +54,52 @@ public void InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
             {
                 bool ignoreCache = true;
                 browserWindow.BrowserControl.Browser.Reload(ignoreCache);
-            };
+            };
+            visCheck += () =>
+            {
+                if (overlayFocused == true && overlayEnabled == true)
+                {
+                    if (browserWindow.IsVisible == false)
+                    {
+                        browserWindow.Show();
+                    }
+                }
+                else
+                {
+                    browserWindow.Hide();
+                }
+            };
+            var currentDispatcher = System.Windows.Threading.Dispatcher.CurrentDispatcher;
             settingsTab.OnToggleWindowVisibility += (o, e) =>
-            {
-                if (browserWindow.IsVisible)
-                    browserWindow.Hide();
-                else
-                    browserWindow.Show();
-            };
+            {
+                overlayEnabled = !overlayEnabled;
+                currentDispatcher.Invoke(visCheck);
+            };
+            System.Windows.Automation.AutomationFocusChangedEventHandler afceh = (sender, o) =>
+            {
+                System.Windows.Automation.AutomationElement focusedElement = sender as System.Windows.Automation.AutomationElement;
+                if (focusedElement != null)
+                {
+                    int processId = focusedElement.Current.ProcessId;
+                    using (System.Diagnostics.Process process = System.Diagnostics.Process.GetProcessById(processId))

```

Comment:
Not knowing what "using" was for, it looked like other pieces of code were using it to save space on null checks, but after looking it up it looks like it's secretly wrapping this with a try/catch block as well, which I don't really care about.  I'll pull this out into another check / early return.